components/org.apache.stratos.manager.console/console/themes/theme0/partials/applications_deploy.hbs

```diff
@@ -22,7 +22,7 @@
 <div id="centered">
     <div class="row title">
         <div class="title-main text-center">
-            <h1>Create Deployment Policy for Application {{applicationAlias}}</h1>
+            <h1>Deploy Deployment Policy for Application {{applicationAlias}}</h1>
```

Comment:
IMO this text needs to be changed to "Deploy Application", we might not need to add the application alias as it has no specific usage at the moment.
Thanks